src/main/java/tlylt/haha/Parser.java

```diff
@@ -0,0 +1,209 @@
+package tlylt.haha;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.Arrays;
+
+/**
+ * Utility class to parse user input.
+ */
+public class Parser {
+    /**
+     * Returns task number.
+     *
+     * @param command String to be parsed.
+     * @return Correct task number as an integer.
+     * @throws HahaTaskNumberNotIntException Task number given is not an integer.
+     */
+    static int taskNumber(String command) throws HahaTaskNumberNotIntException {
```

Comment:
Perhaps this method could be named getTaskNumber, which is a verb, to better represent the method?